CONTRIBUTING.md

```diff
@@ -164,21 +474,32 @@ New committers are approved by majority vote of the TSC ([see TSC charter](https
         * Use the imperative mood in the subject line
         * Wrap the body at 72 characters
         * Use the body to explain what and why vs. how
-     
-- Example commit message
-
-```
-Fix OOM caused by foo in bar
-
-Foo was pack ratting ByteBuffers causing OOM.
-
-Cherry-pick of c2bf3b0 (https://github.com/foo/bar/pull/123/commits/c2bf3b025ff456921c4c5ff4bcb814740e823061)
-
-Co-authored-by: Foo Bar <foo@bar.com>
-```
+    * Ensure all code is peer reviewed within your own organization or peers before submitting
+    * Implement and/or address existing feedback before requesting further review
```

Comment:
Avoid slashes. See [GitLab doc recommended word list entry for slashes](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#slashes) for advice and suggestions.